### DISCUSSION TITLE
Code analyzers for xUnit

### DISCUSSION THREAD

**Discussion by lorcQc** [CommunityVB/Main#9] [General]
@DualBrain 
Hello,
I am working on source code analyzers and code fixes for xUnit.net for VB since they [don't want to support VB](https://github.com/xunit/xunit.analyzers/pull/129#issuecomment-532378250) in the main repository.
Would you be interested in adding this repository in this organization?

**Comment by InteXX** [comment]
This sounds interesting. You've got my vote.

**Comment by DualBrain** [comment]
Oddly I never saw this discussion.... yes, sounds great. If you already have a repo, let me know where to find it and we can add a link to it immediately and then discuss how to proceed.

**Comment by lorcQc** [comment]
@DualBrain 

I put the project on hold for the last few months but I'm interested in coming back to it.

At the moment it is not published on github but I might put it in my personal repo. I just need a few days to clean up the project.

What I'd really like to do is to put it in a community repository, a bit like the VB equivalent of [fsprojects](https://github.com/fsprojects), and finish it with the help of the community

If you're interested, I'd be happy to [transfer it](https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository) to this organization and work on it as an [outside collaborator](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/adding-outside-collaborators-to-repositories-in-your-organization).

**Reply by DualBrain** [reply]
I'm totally on board with this. Just let me know what I need to do on this end to get it under CommunityVB.

**Reply by lorcQc** [reply]
Just create a new repo, named something like CommunityVB.xUnit, and send me an invitation to become a contributor.
I will have to be assigned at least the "write" role.
